parsing_scripts/load_initial_depends.py

```diff
@@ -0,0 +1,69 @@
+import os, json
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "MacPorts.settings")
+
+import django
+
+django.setup()
+
+from ports.models import Dependency, Port
+
+
+def load_depends(list_of_dependencies, type_of_dependency, port):
+    dependency = Dependency()
+    dependencies = []
+    dependency.type = type_of_dependency
+    dependency.port_name = port
+    for depends in list_of_dependencies:
+        dependencies.append(Port.objects.get(name__exact=depends.rsplit(':', 1)[-1]))
+    dependency.save()
+    dependency.dependencies.add(*dependencies)
+
+
+with open('portindex.json', "r") as file:
+    ports = json.load(file)
+
+i = 0
+fixtures = []
+for port in ports:
+    iport = Port.objects.get(name__iexact=port['name'])
```

Comment:
Can we find a better name? The `iport` doesn't tell me anything and makes it somewhat more difficult to understand the code.

Btw: what happens if the database fails to find that port?